dev-python/pyzmq/files/pyzmq-17.1.0-test_message.patch

```diff
@@ -0,0 +1,22 @@
+Disable test failing with python 3.6.
+Bug: https://github.com/zeromq/pyzmq/issues/1202
+
+Patch by Marius Brehler <marbre@linux.sungazer.de>
+--- pyzmq-17.1.0/zmq/tests/test_message.py
++++ pyzmq-17.1.0/zmq/tests/test_message.py
+@@ -14,6 +14,7 @@ import time
+ from pprint import pprint
+ from unittest import TestCase
+ 
++import pytest
+ import zmq
+ from zmq.tests import BaseZMQTestCase, SkipTest, skip_pypy, PYPY
+ from zmq.utils.strtypes import unicode, bytes, b, u
+@@ -262,6 +263,7 @@ class TestFrame(BaseZMQTestCase):
+             self.assertEqual(s,r)
+         self.assertEqual(s, m.bytes)
+     
++    @pytest.mark.skipif(sys.version_info == (3,6))
```

Comment:
FYI:

```
>>> import sys
>>> sys.version_info
sys.version_info(major=3, minor=6, micro=6, releaselevel='final', serial=0)
>>> sys.version_info == (3,6)
False
>>> sys.version_info == (3,6,6)
False
>>> sys.version_info == (3,6,6,'final',0)
True

Reply:
Thanks. Will rework the patch in some way.